### PR TITLE
add an example for using oneAPI AI container

### PR DESCRIPTION
**Pull request by louie-tsai**
# Adding a New Sample(s)

## Description

adding an getting started sample for oneAPI AI Kit container

## Checklist
Administrative
- [ ] Review sample design with the appropriate [Domain Expert](https://github.com/oneapi-src/oneAPI-samples/wiki/Reviewers-and-Domain-Experts): <insert Name Here>
- [ ] If you have any new dependencies/binaries, inform the oneAPI Code Samples Project Manager: @JoeOster

Code Development
- [x] Implement coding guidelines and ensure code quality. [see wiki for details](https://github.com/oneapi-src/oneAPI-samples/wiki/General-Code-Guidelines)
- [x] Adhere to readme template 
- [ ] Enforce format via clang-format config file
- [ ] Adhere to sample.json specification. https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-specification
- [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
- [ ] Run jsonlint on sample.json to verify json syntax. www.jsonlint.com

Security and Legal
- [ ] OSPDT Approval (see @JoeOster for assistance)
- [ ] Compile using the following compiler flags and fix any warnings, the falgs are: "/Wall -Wformat-security -Werror=format-security"
- [ ] Bandit Scans (Python only)
- [ ] Virus scan

Review
- [ ] Review DPC++ code with Paul Peterseon. (GitHub User: pmpeter1)
- [ ] Review readme with Tom Lenth(@tomlenth) and/or Joe Oster(@JoeOster)
- [ ] Tested using Dev Cloud when applicable

### CODE REVIEW

**Comment by louie-tsai** [comment]
@praveenkk123 
Could you help me to look into this PR?

thanks

**Review by tomlenth — APPROVED** [review]

**Comment by louie-tsai** [comment]
@tomlenth 
Addressed the feedback in the PR.
also thanks for Anton's help.

**Review by tomlenth — APPROVED** [review]

**Comment by louie-tsai** [comment]
@tomlenth any further modification needed?

File: AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/README.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  Jupyter notebook and TensorBoard.
+        
+
+## Run the Sample
+
+This sample uses a configuration script to automatically configure the
+environment. This provides fast and less error prone setup. For
+complete instructions for using the oneAPI AI Analytics containers see
+the [Getting Started Guide]
+(https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top/using-containers.html.)
```
Comment (anton-intel) [inline AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/README.md:60]: Please delete a space between ] and (.

[Getting Started Guide](https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top/using-containers.html.)

Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/README.md]: fixed it. thanks for identifying the problem.

**Review by anton-intel — APPROVED** [review]
Looks good to me.
@ishaghosh27, what do you think?

File: AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/run_oneapi_docker.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    exit -1
+fi
+
+image_id="$1"
+name="aikit_container"
+gpu_arg=""
+GPU_DEV=/dev/dri
+if [ -d "$GPU_DEV" ]; then
+    echo "$GPU_DEV exists."
+    gpu_arg=" --device=/dev/dri --ipc=host "
```
Comment (louie-tsai) [inline AI-and-Analytics/Getting-Started-Samples/IntelAIKitContainer_GettingStarted/run_oneapi_docker.sh:13]: might need to apply --ipc=host even on CPU.

**Review by ishaghosh27 — APPROVED** [review]

**Pull request merged**